test/print_env_info.py

```diff
@@ -0,0 +1,373 @@
+# This script outputs relevant system environment info
+# Run it with `python print_env_info.py`.
+from __future__ import absolute_import, division, print_function, unicode_literals
+import locale
+import re
+import subprocess
+import sys
+import os
+from collections import namedtuple
+
+try:
+    import torch
+    TORCH_AVAILABLE = True
+except (ImportError, NameError, AttributeError):
+    TORCH_AVAILABLE = False
+
+# System Environment Information
+SystemEnv = namedtuple('SystemEnv', [
```

Comment:
Name it torch_env per py naming convention . In namedtuple as well.

Reply:
Done.